### PR TITLE
Foundation tab

### PR DESCRIPTION
**Pull request by bitstorm**
Added tab component for Foundation integration

### CODE REVIEW

**Comment by bitstorm** [comment]
@iluwatar 
Hi,

I've added tab component. Please review and tell me if something can be improved.

**Comment by iluwatar** [comment]
Sorry for the delay in reviewing this change, I've been somewhat busy.

**Comment by iluwatar** [comment]
Usually I have added rendering test for the sample pages also. Could you add rendering test for TabPage too?

File: jdk-1.7-parent/wicket-foundation/wicket-foundation-samples/src/main/java/org/wicketstuff/foundation/TabPage.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+</pre>
+				
+				<div>
+					<h4>Rendered HTML</h4>		
+					<div wicket:id="tabHorizontal">
+					</div>
+				</div>
+			</div>
+			<hr>
+			<h2 id="basic">Horizontal Tabs</h2>
```
Comment (iluwatar) [inline jdk-1.7-parent/wicket-foundation/wicket-foundation-samples/src/main/java/org/wicketstuff/foundation/TabPage.html]: Should read: Vertical Tabs

**Comment by iluwatar** [comment]
It would be great to have simple tests for FoundationTab and AjaxFoundationTab components too.

**Comment by iluwatar** [comment]
I've been maintaining this silly counter on CatalogPage.html reading something like:

<h3>41/59 pages (69%) implemented (<a href="http://foundation.zurb.com/" target="blank">Foundation 5.5.1</a>)</h3>

It would be nice to either keep it up to date or then remove it altogether.

File: jdk-1.7-parent/wicket-foundation/wicket-foundation-samples/src/main/java/org/wicketstuff/foundation/TabPage.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+				
+				<h4>HTML</h4>
+
+				<pre><code class="language-html"><div class="code-container"><span class="tag">&lt;<span class="title">div</span> <span class="attribute">wicket:id</span>=<span class="value">&quot;tabHorizontal&quot;</span>&gt;</span><span class="tag">&lt;/<span class="title">div</span>&gt;</span>
+</div></code>
+</pre>
+
+				<h4>Java</h4>
+
+				<pre><code class="language-html"><div class="code-container">add(new FoundationButton("btn", Model.of("Click")));
```
Comment (iluwatar) [inline jdk-1.7-parent/wicket-foundation/wicket-foundation-samples/src/main/java/org/wicketstuff/foundation/TabPage.html]: Should be something like `add(new FoundationTab("tabHorizontal", tabs));`

**Comment by iluwatar** [comment]
JavaDoc for classes `FoundationTab` and `AjaxFoundationTab` would be great. Simple explanation what the classes do.

**Comment by bitstorm** [comment]
Thank you for your suggestions/indications. I will integrate them asap.

**Comment by bitstorm** [comment]
Done. Now it should be ok. Let me know if there's something else we can improve.

**Comment by iluwatar** [comment]
Looks perfect. Great job! :)

**Pull request merged**